### ISSUE TITLE
psc-package init fails with purescript 0.12

### ISSUE THREAD

**Issue by gcsolaroli** [purescript/psc-package#113]
I was trying to follow along Justin's instructions listed [here](https://qiita.com/kimagure/items/570e6f2bbce5b4724564)

Everything looks in order:
```
$ purs --version
0.12.0
$ pulp --version
Pulp version 12.2.0
purs version 0.12.0 using /Users/giulio.cesare/.nvm/versions/node/v10.2.1/bin/purs
$ psc-package --version
0.3.2
```

But when I try to `init` a project I get this logs:
```
$ pulp --psc-package init
* Generating project skeleton in /Users/giulio.cesare/Workarea/personal/test
Initializing new project in current directory
Using the default package set for PureScript compiler version 0.12.0
(Use --source / --set to override this behavior)
warning: Could not find remote branch psc-0.12.0 to clone.
fatal: Remote branch psc-0.12.0 not found in upstream origin
* ERROR: Subcommand terminated with exit code 1
```

Am I doing something wrong, or is there something to be fixed?

**Comment by kritzcreek** [comment]
There is no psc-0.12.0 tag yet, because Justin wants to wait for more than core and contrib to be included in package-sets before making tagging the default for 0.12.0.

The latest available package-set as of right now is: psc-0.12.0-20180528

You can find a listing of all the Tags here: https://github.com/purescript/package-sets/releases

Make a `psc-package.json` file that reads something like:


```
{
    "name": "my-project",
    "set": "psc-0.12.0-20180528",
    "source": "https://github.com/purescript/package-sets.git",
    "depends": [
        "prelude"
    ]
}
```

**Comment by gcsolaroli** [comment]
@kRITZCREEK thanks for the kind response; I am just getting started with these tooling options and I was pretty sure I was missing something. :)
